Uno.Gallery/Uno.Gallery.Shared/Views/SamplePages/GridSamplePage.xaml

```diff
@@ -124,24 +124,24 @@
 						<smtx:XamlDisplay UniqueKey="GridSamplePage_Fluent_Form"
 										  smtx:XamlDisplayExtensions.Header="Form example">
 
-							<Grid HorizontalAlignment="Left" ColumnSpacing="30" RowSpacing="30">
-
-								<Grid.ColumnDefinitions>
-									<ColumnDefinition Width="300"/>
-									<ColumnDefinition Width="300"/>
-								</Grid.ColumnDefinitions>
-
+							<Grid>
 								<Grid.RowDefinitions>
-									<RowDefinition Height="75"/>
-									<RowDefinition Height="75"/>
+									<RowDefinition Height="Auto"/>
+									<RowDefinition Height="*"/>
+									<RowDefinition Height="*"/>
```

Comment:
Looks good! The only thing I would change is removing one of the RowDefinitions and making them all Auto. It will work the same, but look better in the displayed xaml 

<details>
<summary>Screenshot with output</summary>

![image](https://github.com/unoplatform/Uno.Gallery/assets/78549750/68287620-cdad-47c6-858f-3f1160e440f1)

</details>

```suggestion
									<RowDefinition Height="Auto"/>
```